tests/ttnn/unit_tests/test_reshape.py

```diff
@@ -324,3 +324,39 @@ def test_reshape_host(input_shape, output_shape, device):
     output = ttnn.to_torch(ttnn_output)
 
     assert_with_pcc(torch_result, output, 0.9999)
+
+
+@pytest.mark.parametrize(
+    "reshape_pair",
+    [
+        ([((12), (1, 1, 1, 12)), ((32), (1, 1, 1, 32)), ((1, 7), (1, 1, 1, 7))]),
+        ([((12), (1, 1, 1, 12)), ((5), (1, 1, 1, 5)), ((32), (1, 1, 1, 32)), ((1, 7), (1, 1, 1, 7))]),
+        ([((12), (1, 1, 1, 12)), ((32), (1, 1, 1, 32)), ((5), (1, 1, 1, 5)), ((1, 7), (1, 1, 1, 7))]),
+        (
+            [
+                ((12), (1, 1, 1, 12)),
+                ((32), (1, 1, 1, 32)),
+                ((32), (1, 1, 1, 32)),
+            ]
+        ),
+    ],
+)
+@pytest.mark.parametrize(
+    "dtype",
+    [torch.float16, torch.float32],
+)
+def test_reshape_mlir_multiple_reshape(reshape_pair, dtype, device):
+    for pair in reshape_pair:
+        input_shape = pair[0]
+        output_shape = pair[1]
+        torch_input_tensor = torch.randn(input_shape, dtype=dtype)
+        torch_result = torch_input_tensor.reshape(output_shape)
+
+        input_tensor = ttnn.from_torch(
+            torch_input_tensor, layout=ttnn.TILE_LAYOUT, device=device, memory_config=ttnn.DRAM_MEMORY_CONFIG
+        )
```

Comment:
This only makes a single reshape for a given tensor.
Is this intentional?

I thought the idea was to test 
torch
from torch
ttnn reshape
ttnn reshape